Source/Core/Core/IOS/SDIO/SDIOSlot0.cpp

```diff
@@ -289,6 +324,9 @@ s32 SDIOSlot0::ExecuteCommand(const Request& request, u32 _BufferIn, u32 _Buffer
     Memory::Write_U32(0x900, _BufferOut);
     break;
 
+  // The following events don't seem to be possible due to the command length being only 6 bits
+  // I don't want to delete this without more context on why it's there and what it's meant to do
+  /*
   case EVENT_REGISTER:  // async
```

Comment:
I was not aware of the WiiBrew page, although it hardly seems to describe the actual behavior.
I did look at the libogc code, but it seems to be unaware of events. Or perhaps I'm looking in the wrong place.

I'll play around a bit.